core/templates/dev/head/base.js

```diff
@@ -210,6 +212,23 @@ oppia.controller('Base', [
       }, 150);
       return false;
     };
+    // The following methods are defined for the createExplorationButton
```

Comment:
@hitesh96db's approach (creating a separate directive) here is better. I think he's almost done, though, so maybe leave this bit for now until he's merged his PR in?